### PR TITLE
Nodocwrite

### PR DESCRIPTION
**Pull request by plup2**

### CODE REVIEW

File: docs/guide-decl-gcintranet-fr.html [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+		<section class="panel panel-default mrgn-tp-lg">
+			<header class="panel-heading">
+				<h2 id="CDTS_implementation" class="panel-title">Implementation SGDC</h2>
+			</header>
+			<div class="panel-body">
+				<section>
+					<h3 class="h5 mrgn-tp-0">Liste de vérification GCIntranet normal</h3>
+					<ul>
+						<li><a href="#GCIntranet_.2F_version_.2F_default">Version</a></li>
+						<li><a href="#GCIntranet_.2F_version_.2F_default">run</a></li>
```
Comment (j-paquette) [inline docs/guide-decl-gcintranet-fr.html]: @plup2 href should point to `<li><a href="#GCIntranet_.2F_run_.2F_default">run</a></li>`

File: docs/internet-decl-fr.html [hunk 2]
```diff
@@ -0,0 +1,968 @@
...
+		<section>
+			<h3 id="Contr.C3.B4le_de_versions_ou_.C2.AB_run_.C2.BB">Contrôle de versions ou «&#160;run&#160;»</h3>
+			<p>À plusieurs endroits dans ce guide, vous aurez deux options&#160;: avec contrôle de versions ou «&#160;run&#160;».</p>
+			<ul>
+				<li>Le contrôle de version signifie que vous ne recevrez PAS de mise à jour automatique de la BOEW. Vous mettrez en place une «&#160;version&#160;» dans votre actif Web et ajouterez un mécanisme pour la garder à jour pendant que vous faites vos essais pour chaque nouvelle version de la BOEW. Nous suggérons que vous utilisiez votre fichier web.config pour stocker votre variable de version. À EDSC, mettre à jour le fichier web.config en production est beaucoup plus simple que de faire une relâche complète. Cela permet aux actifs Web de demeurer à jour plus rapidement en relâchant seulement le fichier web.config lorsque l’actif Web a été testé et qu’il fonctionne avec la nouvelle version de la BOEW. Des exemples sont fournis pour nos <a href="/wiki/Centrally_Deployed_Templates_Solution_(CDTS)/GoC_.NET_template_guide">gabarits .NET et JAVA (en anglais seulement)</a>.</li>
```
Comment (j-paquette) [inline docs/internet-decl-fr.html]: @plup2: href should point to: `<a href="./templates-fr.html">`

**Review by j-paquette — CHANGES_REQUESTED** [review]

File: docs/guide-decl-gcintranet-fr.html (hunk 1 above)
Comment (plup2) [reply docs/guide-decl-gcintranet-fr.html]: Done

File: docs/internet-decl-fr.html (hunk 2 above)
Comment (plup2) [reply docs/internet-decl-fr.html]: Done

**Review by j-paquette — APPROVED** [review]

**Pull request merged**